### DISCUSSION TITLE
[일지] 5월 28일 (금) 오전 회의록

### DISCUSSION THREAD

**Discussion by areum514** [bcaitech1/p4-opt-5-vibrhanium-#15] [일지]
### 아침계획(오늘계획)
`명환` : src/module 구조 확인
`종헌` : 노래준비
`건우` : 데일리 미션, 미션 34
`재경` : src/module 구조 확인, 데일리 미션
`아름` : 데일리 미션, Knowlege distillation
`재영` : 데일리 미션!

### 멘토님께 질문 
1. autoML로 신경망을 구성할 때 기본 구조는 어떻게 설계되나요?
2. batch size, 모델 구조, loss, lr_scheduler, optimizer등 어떤 걸 우선적으로 찾으시는지 궁금합니다 : ) 
3. 멘토님이 경량화 대회 참여하신다면 어떤 전략을 취하실 것 같으신가요?
4. (후보) 현업에서도 AutoML로 모델을 만들어서 사용하는 경우가 있나요?
5. (후보) Sample data를 뽑아서 실험에 사용하고 싶은데, sample data를 뽑으시는 방법이 있으신가요?
    - ex. train data와 동일한 Category별 분포를 유지하며, radom으로 추출
6. 하이퍼 클로바(hyperCLOVA)에 대한 현업에 있는 분의 생각을 듣고 싶습니다.

### 종헌 신청곡
유리상자 - 문이 열리네요, 라붐 - 상상더하기, 박정현 - 꿈에
